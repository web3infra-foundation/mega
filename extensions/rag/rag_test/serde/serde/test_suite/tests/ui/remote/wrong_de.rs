use serde_derive::Deserialize;

mod remote {
    pub struct S(pub u16);
}

#[derive(Deserialize)]
#[serde(remote = "remote::S")]
struct S(u8);

fn main() {}
