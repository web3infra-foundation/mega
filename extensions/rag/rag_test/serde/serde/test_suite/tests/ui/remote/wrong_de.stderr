error[E0308]: mismatched types
 --> tests/ui/remote/wrong_de.rs:7:10
  |
7 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^ expected `u16`, found `u8`
8 | #[serde(remote = "remote::S")]
  |                  ----------- arguments to this struct are incorrect
  |
note: tuple struct defined here
 --> tests/ui/remote/wrong_de.rs:4:16
  |
4 |     pub struct S(pub u16);
  |                ^
  = note: this error originates in the derive macro `Deserialize` (in Nightly builds, run with -Z macro-backtrace for more info)
